packages/strapi/lib/middlewares/cron/defaults.json

```diff
@@ -1,5 +1,5 @@
 {
   "cron": {
-    "enabled": false
+    "enabled": true
```

Comment:
Not sure if we should enable it by default

Reply:
this doesn't enable the cron by default just runs the middleware init that checks for the server.cron variable that is false by default